ujit.rb

```diff
@@ -35,4 +35,28 @@ def self.disasm(iseq)
 
     str
   end if defined?(Disasm)
+
+  def self.stat_counters
+    Primitive.runtime_counters_to_hash
```

Comment:
Would like some comments explaining what the new methods do, why we're calling runtime_counters_to_hash. Think of Ruby newbies like me, and potential newcomers to the team.